src/parser/shaman/elemental/modules/Abilities.js

```diff
@@ -86,6 +92,21 @@ class Abilities extends CoreAbilities {
         castEfficiency: {
           suggestion: true,
           recommendedEfficiency: 1.0,
+          extraSuggestion: '',
```

Comment:
Are these lines of any use? Considering there's a line in core CastEfficiency that basically ignores this anyway (uses `extraSuggestion` and if it doesn't exist, uses empty string):
https://github.com/WoWAnalyzer/WoWAnalyzer/blob/e4c42eb232b2f100f4a98b362fa20b4baa375851/src/parser/shared/modules/CastEfficiency.js#L294

Reply:
fixed